MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -32,6 +34,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+
+		viewModel?.onErrorStateChange = { [weak self] message in
+			if let message = message {
+				self?.errorView?.show(message: message)
+			} else {
+				self?.errorView?.hideMessage()
+			}
```

Comment:
No need to put question marks after `errorView` as it is has force unwrapped type (`ErrorView!`), please remove the question marks

```suggestion
		viewModel?.onErrorStateChange = { [weak self] message in
			if let message = message {
				self?.errorView.show(message: message)
			} else {
				self?.errorView.hideMessage()
			}
```

Reply:
Done